nix_bisect/nix.py

```diff
@@ -65,8 +65,10 @@ def build_dry(drvs, nix_options=()):
             cur = to_build
         elif line.startswith("/nix/store"):
             cur += [line]
+        elif line.startswith("warning:"):
+            pass
```

Comment:
I think we should pass those warnings on.

```suggestion
            print(f"dry build: {line}", file=sys.stderr)
```

Reply:
Fixed.